### PR TITLE
putting all clones into checkout.sh script and removing them from README

### LINKED ISSUE #132: checkout.sh or README needs updating
If you follow the README step-by-step, you will attempt to clone the soca-bundle twice as it's cloned in the checkout.sh script and in step 0.  We should remove one or the other. Preference on which one to remove? @guillaumevernieres  @flampouris    I can make a PR with whichever removed or if someone has one coming up they can add that to theirs.

### PR DESCRIPTION
**Pull request by JessicaMeixner-NOAA**
## Description

This unifies all cloning of dependent repositories into the checkout.sh script and removes them from the README

### Issue(s) addressed

- fixes #132 

## Testing

Just copy/pasting from README.md, did not test

## Dependencies

None

### CODE REVIEW

**Review by jkbk2004 — APPROVED** [review]
This is right!

**Comment by guillaumevernieres** [comment]
@JessicaMeixner-NOAA looks like some conflicts between this branch and develop ...

**Comment by flampouris** [comment]
Were the conflicts resolved? Should I restore the branch?

**Comment by jkbk2004** [comment]
@flampouris @guillaumevernieres @JessicaMeixner-NOAA An idea is that I can work on resolving this conflict, once feature/auto_test is merged in. This is because I have to update test.setup_godas script accordingly to the change in checkout.sh and readme.md.

**Comment by guillaumevernieres** [comment]
We need to revert the last merge ... I don't think you need to revive the old branch.
Sorry, I should have blocked the pr because of the conflict, but please, @jkbk2004 don't merge!

**Comment by JessicaMeixner-NOAA** [comment]
I'm really confused because the updates in this PR are in the master... https://github.com/NOAA-EMC/godas/commit/c7948f18ca7c7db3096351b87cba1bb16d45460a so I'm assuming conflicts were resolved one way or another.

**Pull request closed without merging**
